src/SonarScanner.MSBuild.Tasks/MSBuildLoggerAdapter.cs

```diff
@@ -68,6 +68,16 @@ void ILogger.WriteUIWarnings(string outputFolder)
         // no-op
     }
 
+    void ILogger.AddTelemetryMessage(string key, string value)
```

Comment:
These are lowering the coverage.
I still think we should separate analysis warning and our telemetry from the ILogger for this exact reason here. :/ 
But in another sprint.
If you think I will add a ticket with all the details.